tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/ProgrammeMembershipResourceIntTest.java

```diff
@@ -330,6 +330,10 @@ void shouldGetProgrammeMembershipSummaryListByUuid2() throws Exception {
     restProgrammeMembershipMockMvc.perform(get("/api/programme-memberships/summary-list")
             .param("ids", "invalid-uuid"))
         .andExpect(status().isBadRequest());
+
+    restProgrammeMembershipMockMvc.perform(get("/api/programme-memberships/summary-list")
+            .param("ids", ""))
+        .andExpect(status().isOk());
```

Comment:
I think we need to be explicit about what the content should be under these circumstances, not just the status